assets/scss/components/_hero.scss

```diff
@@ -133,6 +133,7 @@
         margin-top: spacing($size: 12, $negative: true);
 
         .text-animation__text {
+          word-break: break-all;
```

Comment:
Nutz lieber word-wrap: break-word; damit kannst du dann auch mit soll bruchstellen wie &shy; arbeiten und könntest dann anstatt "Berufseinsteiger" den Text "Beruf&shy;se&shy;in&shy;steiger" pflegen oder vermutlich reicht auch "Berufsein&shy;steiger"